app/views/elements/evaluations/mixeval_eval_form.ctp

```diff
@@ -51,13 +51,26 @@
         <h1 id="peer-title" class="title"><?php __("Peer Evaluation Questions")?></h1>
 
         <?php
+        array_multisort($questions);
```

Comment:
Is the `array_multisort` useful considering the below sorting being applied?

Reply:
In this context might not have any impact on the data returning from the database, but generally it is a good practice 'I think' to get the initial data (multi-dimensional-array) sorted by the first key values which is not applicable here since the initial data is structured differently.

I left it in case i need it in another context.

Please remove it/comment it out or let me know if you want me to push a modified copy.

Cheers.

Reply:
If its unneeded, please remove it from your commit (you can amend the commit).

Reply:
Done. 
array_multisort is removed.